chef/cookbooks/apache2/recipes/mod_ssl.rb

```diff
@@ -40,6 +40,7 @@
   # override the resource defined in default.rb; we don't want to create the
   # resource again, otherwise we will write the file twice
   resource = resources(template: "#{node[:apache][:dir]}/ports.conf")
+  node[:apache][:listen_ports] << "443"
```

Comment:
This appears to be trying to append the 443 entry to the listen_ports list/array...
But on the next line you are "combining" 443 with the listen_ports list value in a different way... Which may not longer be required now if you have appended 443 into the listen_ports list...